tabpy-server/tabpy_server/common/endpoint_file_mgr.py

```diff
@@ -14,7 +14,7 @@
 import time
 from re import compile as _compile
 
-from tabpy_server.common.tabpy_logging import (
+from tabpy_tools.tabpy_logging import (
```

Comment:
It's not explicitly stated in the setup but by importing like this you are essentially requiring that the client and server be installed together, which I don't think you really want to require (?)

Reply:
Client needs server and server needs client due to how the things are implemented right now :( But at least we remove tabpy-server package with this change.

Reply:
Yea I think the logging was the only linkage between the two though. I should have time later this week to merge in this change and resolve conflicts so should be able to resolve the circular dependency thereafter